formfactory/views.py

```diff
@@ -22,6 +24,24 @@ def __init__(self, *args, **kwargs):
 
     def get_template_names(self):
         template_names = []
+        template = self.kwargs.get("template", None)
```

Comment:
`template` is currently unused. But consider how it can be used for the suggestion above regarding `render_form`